app/src/main/res/drawable/ic_baseline_add_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#5807A5"
```

Comment:
Can you please rename this file to something more readable please Hellin? 😄

Reply:
it is now changed on my latest pull request, thanks!